src/libfreenect2.cpp

```diff
@@ -33,6 +33,10 @@
 #include <limits>
 #include <cmath>
 #include <cstdlib>
+#include <dirent.h>
+#include <iostream>
```

Comment:
I do not see `iostream` being used in the new code. Remove.

Reply:
(Removed)

Reply:
DONE